nautobot_chatops/dispatchers/base.py

```diff
@@ -37,6 +38,38 @@ def __init__(self, context=None):
         """Init this Dispatcher with the provided dict of contextual information (which will vary by app)."""
         self.context = context or {}
 
+    def _get_cache_key(self) -> str:
+        """Key generator for the cache, adding the plugin prefix name."""
+        # Using __file__ as a key customization within the cache
+        key_string = "-".join([__file__, self.context.get("user_name")])
```

Comment:
Might be safer to use `user_id` rather than `user_name` here as I know the former should be unique, but the latter might not be guaranteed to be.